templates/search_results.hbs

```diff
@@ -15,7 +15,7 @@
     <aside class="search-results-sidebar">
       {{#if source_filters}}
         <section class="filters-in-section collapsible-sidebar" aria-expanded="false">
-          <button type="button" class="collapsible-sidebar-toggle" aria-expanded="false">
+          <button aria-label="{{t 'search_result_source_menu'}}" type="button" class="collapsible-sidebar-toggle" aria-expanded="false">
```

Comment:
Nitpick: I know we don't have a consistent guideline for this, but it might be easier to read the code in the future if we order attributes https://codeguide.co/#html-attribute-order
<button class="collapsible-sidebar-toggle" type="button"  aria-expanded="false" aria-label="{{t 'search_result_source_menu'}}">